### PR TITLE
update phpcs to 3.0

### PR DESCRIPTION
**Pull request by h-bragg**
Update phpcs to v3

Enables cache and parallel processing of files.

### CODE REVIEW

**Comment by h-bragg** [comment]
see graze/web#5831

**Comment by h-bragg** [comment]
still not fulling released... will leave un-merged until it is

**Comment by h-bragg** [comment]
3.0 has been fully released and cli cache commands are working correctly.

File: composer.json [hunk 1]
```diff
@@ -10,9 +10,12 @@
...
-            "name": "Will Pillar",
-            "email": "will.pillar@graze.com"
+            "name": "Graze Developers",
+            "email": "developers@graze.com"
         }
     ],
-    "require": {}
+    "require": {},
+    "require-dev": {
+        "squizlabs/php_codesniffer": "^3.0"
```
Comment (biggianteye) [inline composer.json]: What was the reason behind including this? It doesn't seem to be used.

There is some irony in the fact that the code in this repo doesn't conform to our code standards. 😆  I think that should be fixed but that can probably be left to another branch.

Comment (h-bragg) [inline composer.json]: i think i ran composer locally so i could check in phpstorm.

Thats probably a good point, maybe i'll setup some installation and linting

**Comment by h-bragg** [comment]
@biggianteye added building and linting w/ travis

**Comment by h-bragg** [comment]
@biggianteye went and did some other bits that spawned from a re-look.

namely committing .lock files because of https://yarnpkg.com/blog/2016/11/24/lockfiles-for-all/

**Pull request merged**